commons/c2cgeoportal_commons/models/main.py

```diff
@@ -414,17 +414,6 @@ class LayerGroup(TreeGroup):
             'title': _('Expanded'),
             'column': 2
         }})  # shouldn't be used in V3
```

Comment:
should the comment be removed ?

Reply:
No because it's steel be the case, but we can't remove it because the script themev1tov2 use it :-)